packages/typegpu-geometry/src/circle.ts

```diff
@@ -13,8 +13,8 @@ const SubdivLevelResult = struct({
 
 const getSubdivLevel = tgpu.fn([u32], SubdivLevelResult)(
   (vertexIndex) => {
-    let totalVertexCount = u32(0);
-    for (let level = u32(0); level < 8; level += 1) {
+    let totalVertexCount = u32();
+    for (let level = u32(); level < 8; level += 1) {
```

Comment:
is there a reason to change these two lines? I kinda preferred seeing the explicit 0 in there